### ISSUE TITLE
Disabled attribute doesn't work.

### ISSUE THREAD

**Issue by axljondelacruz** [st-h/ember-content-editable#109]
Hello!

Thanks for making this! Really helps me to not have to write one of my own lol. Though I noticed that when setting `disabled`, the `ember-content-editable` class is still on the element and so the css class is still getting applied.

```
.ember-content-editable {
  cursor: text;
  -webkit-user-modify: read-write-plaintext-only;
}
```

Would possibly help to throw that into `classNameBindings` as `contenteditable:ember-content-editable`?

### Update

I see that this relates to Issue https://github.com/st-h/ember-content-editable/issues/105 and PR #99 . Scoping the CSS would work better in this scenario! 👍

**Comment by axljondelacruz** [comment]
I see that this is somewhat relates to Issue https://github.com/st-h/ember-content-editable/issues/105 and PR #99 . Hope this helps! 👍

**Comment by st-h** [comment]
sorry about the late reply. This should be now fixed in version 3.0.0. Upgrades require to migrate due to two way binding no longer being available with current ember versions.
